api/modules/users/serializers.py

```diff
@@ -10,10 +10,11 @@ class UserSerializer(serializers.ModelSerializer):
     """
     image = serializers.SerializerMethodField()
     status = serializers.SerializerMethodField()
+    unique_code = serializers.SerializerMethodField()
```

Comment:
Remove `unique_code ` from serializer, since you shouldn't be having any changes in model